ImplementationGuide/markdown/Datenobjekte/Operations.md

```diff
@@ -182,16 +185,3 @@ Beispiel: Absage eines Termins
 ### Anlage einer Patient-Ressource
 
 Ein Termin Repository MUSS die Anlage (Create-Interaktion) einer Patient-Ressource entsprechend der Vorgaben des [ISiK-Basismoduls](https://simplifier.net/guide/implementierungsleitfadenisik-basismodul/I-UebergreifendeFestlegungen-UebergreifendeFestlegungen-Rest?version=current) unterstützen.
-
-
----
-
-### Asynchrone Ausführung $book
-
-Die Operation zur Buchung eines Termin MUSS ebenfalls asychron ausgeführt werden können, für den Fall, dass ein Terminrepository keine Terminbestätigung als direkte Antwort versenden kann. Beispielsweise kann dies der Fall sein, falls ein Termin zunächst manuell im Terminrepository bestätigt werden muss. Es gelten die Regeln der [FHIR Kernspezifikation - Abschnitt 3.2.0.7 Executing an Operation Asynchronously](https://www.hl7.org/fhir/r4/operations.html):
```

Comment:
Kurzes Feedback aus einem Termin mit Francois: In den ursprünglichen Meetings zum Design dieser Operation hatten wir von den Herstellern die Rückmeldung erhalten, dass es teilweise erheblich dauern kann bis ein System den Status entscheiden kann. Hier kann es dann passieren, dass man in einen HTTP Timeout läuft. Daher wurde die  asychrone Variante definiert.

Reply:
aber woher weiß denn dann der Client, ob er synchron oder asynchron anfragen muss?

Reply:
Mit der oben vorgeschlagen Änderung wäre es eine bequemere Alternative, da der Client sich die erneute Suche sparen kann. Könnte der Client sich nicht aussuchen welche Variante verwendet wird?

Reply:
d.h. wenn ich den gelöschten Absatz zu der asynchronen Kommunikation wieder reinnehme, passt es? Auch wenn dann das Problem bleibt, dass der CLient Wissen über den Server braucht, um zu wählen, ob er synchron oder asynchron anfragt...